spec/features/admin/legislation/processes_spec.rb

```diff
@@ -75,6 +77,48 @@
       expect(page).to have_content 'Summary of the process'
       expect(page).not_to have_content 'Describing the process'
     end
+
+
+    scenario 'Legislation process in draft phase' do
+      visit admin_root_path
+
+      within('#side_menu') do
+        click_link "Collaborative Legislation"
+      end
+
+      expect(page).not_to have_content 'An example legislation process'
+
+      click_link "New process"
+
+      fill_in 'legislation_process_title_en', with: 'An example legislation process in draft phase'
+      fill_in 'legislation_process_summary_en', with: 'Summary of the process'
+      fill_in 'legislation_process_description_en', with: 'Describing the process'
```

Comment:
This spec it failing right now, could you change these lines to:

```
fill_in 'Process Title', with: 'An example legislation process in draft phase'
fill_in 'Summary', with: 'Summary of the process'
fill_in 'Description', with: 'Describing the process'
```

This change avoid spec fails 😉